requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/canonical/operator/#egg=ops
```

Comment:
This package is on [pypi](https://pypi.org/project/ops/) so you shouldn't have to pull it from github

Reply:
In this case, @DnPlas is using `container.can_connect` which hasn't yet made it into a release, so this is necessary :)

Reply:
Basically that's the reason. Resolving this.